Kickstarter-iOS/Views/Controllers/LiveStreamContainerViewController.swift

```diff
@@ -93,6 +93,7 @@ public final class LiveStreamContainerViewController: UIViewController {
     _  = self.loaderLabel
       |> UILabel.lens.font .~ .ksr_headline(size: 14)
       |> UILabel.lens.textColor .~ .white
+      |> UILabel.lens.textAlignment .~ .center
```

Comment:
This caused almost all the snapshots for this view controller to have to be re-recorded 😐